cmd/server/assets/users/show.html

```diff
@@ -163,26 +153,28 @@ <h5 class="modal-title">Codes issued by {{$user.Name}}</h5>
           dataTable.addRow([utcDate(row.date), row.data.codes_issued]);
         });
 
+        let dateFormatter = new google.visualization.DateFormat({
+          pattern: 'MMM dd',
```

Comment:
I'm not sure the pattern syntax, but I've never seen a three-character `MMM` for month?

Reply:
It's "Nov" "Dec" according to their docs

Reply:
ah